tensorboard/plugins/hparams/api_test.py

```diff
@@ -155,6 +155,63 @@ def test_summary_pb(self):
         expected_experiment_pb,
     )
 
+  def _assert_unique_summary(self, logdir, summary_pb):
+    """Test that `logdir` contains exactly one summary, `summary_pb`.
+
+    Specifically, `logdir` must be a directory containing exactly one
+    entry, which must be an events file of whose events exactly one is a
+    summary, which must be equal to `summary_pb`.
+
+    Args:
+      logdir: String path to a logdir.
+      summary_pb: A `summary_pb2.Summary` object.
+    """
+    files = os.listdir(logdir)
+    self.assertEqual(len(files), 1, files)
+    events_file = os.path.join(logdir, files[0])
+    for event in tf.compat.v1.train.summary_iterator(events_file):
+      if event.WhichOneof("what") != "summary":
+        continue
+      self.assertEqual(event.summary, summary_pb)
+      break
```

Comment:
nit, but you can simplify this to just `return` here and then skip the `else` clause and `self.fail()` unconditionally.  It's one line shorter and avoids introducing a slightly more novel language feature when there's a more standard alternative.